class/defaults.yml

```
@@ -9,7 +9,7 @@ parameters:
       tags: []
       git: {}
     provider: ${facts:cloud}
-    version: master
+    version: v1.0.0
```

It might make sense to provide decoupled default versions for Exoscale and cloudscale.ch, as we don't really want to be bound to keep the TF module versions in lockstep.

```suggestion
    version: ${openshift4_terraform:_tf_module_version:${openshift4_terraform:provider}}
```

And add the following parameter:

```
parameters:
  openshift4_terraform:
    =_tf_module_version:
      cloudscale: v1.0.0
      exoscale: v1.0.0
```

---

that's way more elegant :+1: